{
  "SchemaVersion": 2,
  "CreatedAt": "2025-09-23T09:31:43.257518163Z",
  "ArtifactName": "namelivia/like-and-share-twitter-bot:latest",
  "ArtifactType": "container_image",
  "Metadata": {
    "Size": 283973632,
    "OS": {
      "Family": "alpine",
      "Name": "3.18.2",
      "EOSL": true
    },
    "ImageID": "sha256:ccf7f4af8d05bf5ad2dca899b82e32e7a29d565e18f2bc9a1c5cd440f2bec234",
    "DiffIDs": [
      "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c",
      "sha256:cd18acbc1cce1afc3e8ecd73699c93e81a2ab76778933bf170162f7ef8997702",
      "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588",
      "sha256:3a7334d9cbff6d75b8396da25ad8f79556853720ac0a494172579f650369e1b6",
      "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a",
      "sha256:478a72c0919caff091fae0799bf3a893a4f999653b37a85382672bcaff6d2d4b",
      "sha256:c24514e33d4f7f261e65b710d281707626624cc53b181d22c2d2e60949d107ca",
      "sha256:f6a988e312fc01079acbe1e562c39b411a4cbc23a18bf56b3c1ac537643c8217",
      "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de",
      "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
    ],
    "RepoTags": [
      "namelivia/like-and-share-twitter-bot:latest"
    ],
    "RepoDigests": [
      "namelivia/like-and-share-twitter-bot@sha256:71c7eebffe48af072d43ef35e9ccc4b08174e79a54c6bd7cea08f0e29094b425"
    ],
    "ImageConfig": {
      "architecture": "amd64",
      "created": "2023-08-07T08:22:47.237127956Z",
      "history": [
        {
          "created": "2023-06-14T20:41:58.950178204Z",
          "created_by": "/bin/sh -c #(nop) ADD file:1da756d12551a0e3e793e02ef87432d69d4968937bd11bed0af215db19dd94cd in / "
        },
        {
          "created": "2023-06-14T20:41:59.079795125Z",
          "created_by": "/bin/sh -c #(nop)  CMD [\"/bin/sh\"]",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV LANG=C.UTF-8",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "RUN /bin/sh -c set -eux; \tapk add --no-cache \t\tca-certificates \t\ttzdata \t; # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV GPG_KEY=0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PYTHON_VERSION=3.7.17",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "RUN /bin/sh -c set -eux; \t\tapk add --no-cache --virtual .build-deps \t\tgnupg \t\ttar \t\txz \t\t\t\tbluez-dev \t\tbzip2-dev \t\tdpkg-dev dpkg \t\texpat-dev \t\tfindutils \t\tgcc \t\tgdbm-dev \t\tlibc-dev \t\tlibffi-dev \t\tlibnsl-dev \t\tlibtirpc-dev \t\tlinux-headers \t\tmake \t\tncurses-dev \t\topenssl-dev \t\tpax-utils \t\treadline-dev \t\tsqlite-dev \t\ttcl-dev \t\ttk \t\ttk-dev \t\tutil-linux-dev \t\txz-dev \t\tzlib-dev \t; \t\twget -O python.tar.xz \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz\"; \twget -O python.tar.xz.asc \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc\"; \tGNUPGHOME=\"$(mktemp -d)\"; export GNUPGHOME; \tgpg --batch --keyserver hkps://keys.openpgp.org --recv-keys \"$GPG_KEY\"; \tgpg --batch --verify python.tar.xz.asc python.tar.xz; \tgpgconf --kill all; \trm -rf \"$GNUPGHOME\" python.tar.xz.asc; \tmkdir -p /usr/src/python; \ttar --extract --directory /usr/src/python --strip-components=1 --file python.tar.xz; \trm python.tar.xz; \t\tcd /usr/src/python; \tgnuArch=\"$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)\"; \t./configure \t\t--build=\"$gnuArch\" \t\t--enable-loadable-sqlite-extensions \t\t--enable-optimizations \t\t--enable-option-checking=fatal \t\t--enable-shared \t\t--with-system-expat \t\t--without-ensurepip \t; \tnproc=\"$(nproc)\"; \tEXTRA_CFLAGS=\"-DTHREAD_STACK_SIZE=0x100000\"; \tLDFLAGS=\"${LDFLAGS:--Wl},--strip-all\"; \tPROFILE_TASK='-m test.regrtest --pgo \t\ttest_array \t\ttest_base64 \t\ttest_binascii \t\ttest_binhex \t\ttest_binop \t\ttest_bytes \t\ttest_c_locale_coercion \t\ttest_class \t\ttest_cmath \t\ttest_codecs \t\ttest_compile \t\ttest_complex \t\ttest_csv \t\ttest_decimal \t\ttest_dict \t\ttest_float \t\ttest_fstring \t\ttest_hashlib \t\ttest_io \t\ttest_iter \t\ttest_json \t\ttest_long \t\ttest_math \t\ttest_memoryview \t\ttest_pickle \t\ttest_re \t\ttest_set \t\ttest_slice \t\ttest_struct \t\ttest_threading \t\ttest_time \t\ttest_traceback \t\ttest_unicode \t'; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:-}\" \t\t\"PROFILE_TASK=${PROFILE_TASK:-}\" \t; \trm python; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:--Wl},-rpath='\\$\\$ORIGIN/../lib'\" \t\t\"PROFILE_TASK=${PROFILE_TASK:-}\" \t\tpython \t; \tmake install; \t\tcd /; \trm -rf /usr/src/python; \t\tfind /usr/local -depth \t\t\\( \t\t\t\\( -type d -a \\( -name test -o -name tests -o -name idle_test \\) \\) \t\t\t-o \\( -type f -a \\( -name '*.pyc' -o -name '*.pyo' -o -name 'libpython*.a' \\) \\) \t\t\t-o \\( -type f -a -name 'wininst-*.exe' \\) \t\t\\) -exec rm -rf '{}' + \t; \t\tfind /usr/local -type f -executable -not \\( -name '*tkinter*' \\) -exec scanelf --needed --nobanner --format '%n#p' '{}' ';' \t\t| tr ',' '\\n' \t\t| sort -u \t\t| awk 'system(\"[ -e /usr/local/lib/\" $1 \" ]\") == 0 { next } { print \"so:\" $1 }' \t\t| xargs -rt apk add --no-network --virtual .python-rundeps \t; \tapk del --no-network .build-deps; \t\tpython3 --version # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "RUN /bin/sh -c set -eux; \tfor src in idle3 pydoc3 python3 python3-config; do \t\tdst=\"$(echo \"$src\" | tr -d 3)\"; \t\t[ -s \"/usr/local/bin/$src\" ]; \t\t[ ! -e \"/usr/local/bin/$dst\" ]; \t\tln -svT \"$src\" \"/usr/local/bin/$dst\"; \tdone # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PYTHON_PIP_VERSION=23.0.1",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PYTHON_SETUPTOOLS_VERSION=57.5.0",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/0d8570dc44796f4369b652222cf176b3db6ac70e/public/get-pip.py",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "ENV PYTHON_GET_PIP_SHA256=96461deced5c2a487ddc65207ec5a9cffeca0d34e7af7ea1afc470ff0d746207",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "RUN /bin/sh -c set -eux; \t\twget -O get-pip.py \"$PYTHON_GET_PIP_URL\"; \techo \"$PYTHON_GET_PIP_SHA256 *get-pip.py\" | sha256sum -c -; \t\texport PYTHONDONTWRITEBYTECODE=1; \t\tpython get-pip.py \t\t--disable-pip-version-check \t\t--no-cache-dir \t\t--no-compile \t\t\"pip==$PYTHON_PIP_VERSION\" \t\t\"setuptools==$PYTHON_SETUPTOOLS_VERSION\" \t; \trm -f get-pip.py; \t\tpip --version # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-06-06T23:32:36Z",
          "created_by": "CMD [\"python3\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2023-08-07T08:22:30.972702699Z",
          "created_by": "WORKDIR /app",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-08-07T08:22:31.012070261Z",
          "created_by": "COPY . /app # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-08-07T08:22:33.226385879Z",
          "created_by": "RUN /bin/sh -c apk add gcc musl-dev # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-08-07T08:22:39.473500624Z",
          "created_by": "RUN /bin/sh -c pip install -I pipenv==2022.10.25 # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-08-07T08:22:47.237127956Z",
          "created_by": "RUN /bin/sh -c pipenv install # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2023-08-07T08:22:47.237127956Z",
          "created_by": "CMD [\"pipenv\" \"run\" \"python\" \"main.py\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        }
      ],
      "os": "linux",
      "rootfs": {
        "type": "layers",
        "diff_ids": [
          "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c",
          "sha256:cd18acbc1cce1afc3e8ecd73699c93e81a2ab76778933bf170162f7ef8997702",
          "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588",
          "sha256:3a7334d9cbff6d75b8396da25ad8f79556853720ac0a494172579f650369e1b6",
          "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a",
          "sha256:478a72c0919caff091fae0799bf3a893a4f999653b37a85382672bcaff6d2d4b",
          "sha256:c24514e33d4f7f261e65b710d281707626624cc53b181d22c2d2e60949d107ca",
          "sha256:f6a988e312fc01079acbe1e562c39b411a4cbc23a18bf56b3c1ac537643c8217",
          "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de",
          "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
        ]
      },
      "config": {
        "Cmd": [
          "pipenv",
          "run",
          "python",
          "main.py"
        ],
        "Env": [
          "PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "LANG=C.UTF-8",
          "GPG_KEY=0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D",
          "PYTHON_VERSION=3.7.17",
          "PYTHON_PIP_VERSION=23.0.1",
          "PYTHON_SETUPTOOLS_VERSION=57.5.0",
          "PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/0d8570dc44796f4369b652222cf176b3db6ac70e/public/get-pip.py",
          "PYTHON_GET_PIP_SHA256=96461deced5c2a487ddc65207ec5a9cffeca0d34e7af7ea1afc470ff0d746207"
        ],
        "WorkingDir": "/app",
        "ArgsEscaped": true
      }
    },
    "Layers": [
      {
        "Size": 7621632,
        "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
        "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
      },
      {
        "Size": 2727424,
        "Digest": "sha256:cfcc276e4459ecb909a4257fdf7647034eb3b2ab1f0d0c0bba54856c39079c50",
        "DiffID": "sha256:cd18acbc1cce1afc3e8ecd73699c93e81a2ab76778933bf170162f7ef8997702"
      },
      {
        "Size": 28612608,
        "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
        "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
      },
      {
        "Size": 5120,
        "Digest": "sha256:c41f20535e037179d2cd103657bb7167c1080eca14a754344eab330ada692782",
        "DiffID": "sha256:3a7334d9cbff6d75b8396da25ad8f79556853720ac0a494172579f650369e1b6"
      },
      {
        "Size": 10492416,
        "Digest": "sha256:f053299a814f964803234a48003d7c807f15a04ab5d15c3dac974b2b9ac19f88",
        "DiffID": "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a"
      },
      {
        "Size": 1536,
        "Digest": "sha256:bbae5b1de4f8486540232645941a19fd4a414868d5a5a5c0ae49e83d66e1cf9d",
        "DiffID": "sha256:478a72c0919caff091fae0799bf3a893a4f999653b37a85382672bcaff6d2d4b"
      },
      {
        "Size": 150528,
        "Digest": "sha256:606f09f5cc375fa00c0cd4774c862dbd7ae713270a8ce07f16fcf60e40de275f",
        "DiffID": "sha256:c24514e33d4f7f261e65b710d281707626624cc53b181d22c2d2e60949d107ca"
      },
      {
        "Size": 150478336,
        "Digest": "sha256:316eae1787993dc0746517bc86d7137ce87e10873ac6f92a5d7b19a341c79ed0",
        "DiffID": "sha256:f6a988e312fc01079acbe1e562c39b411a4cbc23a18bf56b3c1ac537643c8217"
      },
      {
        "Size": 46425088,
        "Digest": "sha256:2a88c5c4b5ec4f7721b26e9fdbf73049eb2ec929611bb72deb12b830f452a738",
        "DiffID": "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de"
      },
      {
        "Size": 37458944,
        "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
        "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
      }
    ]
  },
  "Results": [
    {
      "Target": "namelivia/like-and-share-twitter-bot:latest (alpine 3.18.2)",
      "Class": "os-pkgs",
      "Type": "alpine",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2025-0840",
          "PkgID": "binutils@2.40-r7",
          "PkgName": "binutils",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/binutils@2.40-r7?arch=x86_64\u0026distro=3.18.2",
            "UID": "e11225e0381cd527"
          },
          "InstalledVersion": "2.40-r7",
          "FixedVersion": "2.40-r8",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:316eae1787993dc0746517bc86d7137ce87e10873ac6f92a5d7b19a341c79ed0",
            "DiffID": "sha256:f6a988e312fc01079acbe1e562c39b411a4cbc23a18bf56b3c1ac537643c8217"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-0840",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "binutils: GNU Binutils objdump.c disassemble_bytes stack-based overflow",
          "Description": "A vulnerability, which was classified as problematic, was found in GNU Binutils up to 2.43. This affects the function disassemble_bytes of the file binutils/objdump.c. The manipulation of the argument buf leads to stack-based buffer overflow. It is possible to initiate the attack remotely. The complexity of an attack is rather high. The exploitability is told to be difficult. The exploit has been disclosed to the public and may be used. Upgrading to version 2.44 is able to address this issue. The identifier of the patch is baac6c221e9d69335bf41366a1c7d87d8ab2f893. It is recommended to upgrade the affected component.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-119",
            "CWE-121",
            "CWE-787"
          ],
          "VendorSeverity": {
            "amazon": 2,
            "azure": 2,
            "cbl-mariner": 2,
            "nvd": 3,
            "photon": 3,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:L/I:L/A:L",
              "V3Score": 5
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2025-0840",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-0840",
            "https://sourceware.org/bugzilla/attachment.cgi?id=15882",
            "https://sourceware.org/bugzilla/show_bug.cgi?id=32560",
            "https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;h=baac6c221e9d69335bf41366a1c7d87d8ab2f893",
            "https://ubuntu.com/security/notices/USN-7306-1",
            "https://ubuntu.com/security/notices/USN-7423-2",
            "https://vuldb.com/?ctiid.293997",
            "https://vuldb.com/?id.293997",
            "https://vuldb.com/?submit.485255",
            "https://www.cve.org/CVERecord?id=CVE-2025-0840",
            "https://www.gnu.org/"
          ],
          "PublishedDate": "2025-01-29T20:15:35.007Z",
          "LastModifiedDate": "2025-03-04T14:53:43.637Z"
        },
        {
          "VulnerabilityID": "CVE-2022-48174",
          "PkgID": "busybox@1.36.1-r0",
          "PkgName": "busybox",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/busybox@1.36.1-r0?arch=x86_64\u0026distro=3.18.2",
            "UID": "a4df3c9c8356ea9"
          },
          "InstalledVersion": "1.36.1-r0",
          "FixedVersion": "1.36.1-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2022-48174",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "busybox: stack overflow vulnerability in ash.c leads to arbitrary code execution",
          "Description": "There is a stack overflow vulnerability in ash.c:6030 in busybox before 1.35. In the environment of Internet of Vehicles, this vulnerability can be executed from command to arbitrary code execution.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-787"
          ],
          "VendorSeverity": {
            "amazon": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 3,
            "redhat": 3,
            "ubuntu": 1
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2022-48174",
            "https://bugs.busybox.net/show_bug.cgi?id=15216",
            "https://linux.oracle.com/cve/CVE-2022-48174.html",
            "https://linux.oracle.com/errata/ELSA-2023-5178.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2022-48174",
            "https://security.netapp.com/advisory/ntap-20241129-0001/",
            "https://ubuntu.com/security/notices/USN-6335-1",
            "https://ubuntu.com/security/notices/USN-6961-1",
            "https://www.cve.org/CVERecord?id=CVE-2022-48174"
          ],
          "PublishedDate": "2023-08-22T19:16:31.08Z",
          "LastModifiedDate": "2025-02-05T18:02:49.267Z"
        },
        {
          "VulnerabilityID": "CVE-2022-48174",
          "PkgID": "busybox-binsh@1.36.1-r0",
          "PkgName": "busybox-binsh",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/busybox-binsh@1.36.1-r0?arch=x86_64\u0026distro=3.18.2",
            "UID": "b1aaee90be1f5c9f"
          },
          "InstalledVersion": "1.36.1-r0",
          "FixedVersion": "1.36.1-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2022-48174",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "busybox: stack overflow vulnerability in ash.c leads to arbitrary code execution",
          "Description": "There is a stack overflow vulnerability in ash.c:6030 in busybox before 1.35. In the environment of Internet of Vehicles, this vulnerability can be executed from command to arbitrary code execution.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-787"
          ],
          "VendorSeverity": {
            "amazon": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 3,
            "redhat": 3,
            "ubuntu": 1
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2022-48174",
            "https://bugs.busybox.net/show_bug.cgi?id=15216",
            "https://linux.oracle.com/cve/CVE-2022-48174.html",
            "https://linux.oracle.com/errata/ELSA-2023-5178.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2022-48174",
            "https://security.netapp.com/advisory/ntap-20241129-0001/",
            "https://ubuntu.com/security/notices/USN-6335-1",
            "https://ubuntu.com/security/notices/USN-6961-1",
            "https://www.cve.org/CVERecord?id=CVE-2022-48174"
          ],
          "PublishedDate": "2023-08-22T19:16:31.08Z",
          "LastModifiedDate": "2025-02-05T18:02:49.267Z"
        },
        {
          "VulnerabilityID": "CVE-2024-37371",
          "PkgID": "krb5-libs@1.20.1-r1",
          "PkgName": "krb5-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/krb5-libs@1.20.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "8d0773d3c16b905d"
          },
          "InstalledVersion": "1.20.1-r1",
          "FixedVersion": "1.20.2-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-37371",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "krb5: GSS message token handling",
          "Description": "In MIT Kerberos 5 (aka krb5) before 1.21.3, an attacker can cause invalid memory reads during GSS message token handling by sending message tokens with invalid length fields.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-125"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 3,
            "photon": 4,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
              "V3Score": 9.1
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 6.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:1671",
            "https://access.redhat.com/security/cve/CVE-2024-37371",
            "https://bugzilla.redhat.com/2294581",
            "https://bugzilla.redhat.com/2294676",
            "https://bugzilla.redhat.com/2301888",
            "https://bugzilla.redhat.com/2318857",
            "https://bugzilla.redhat.com/2318858",
            "https://bugzilla.redhat.com/2318870",
            "https://bugzilla.redhat.com/2318873",
            "https://bugzilla.redhat.com/2318874",
            "https://bugzilla.redhat.com/2318876",
            "https://bugzilla.redhat.com/2318882",
            "https://bugzilla.redhat.com/2318883",
            "https://bugzilla.redhat.com/2318884",
            "https://bugzilla.redhat.com/2318885",
            "https://bugzilla.redhat.com/2318886",
            "https://bugzilla.redhat.com/2318897",
            "https://bugzilla.redhat.com/2318900",
            "https://bugzilla.redhat.com/2318905",
            "https://bugzilla.redhat.com/2318914",
            "https://bugzilla.redhat.com/2318922",
            "https://bugzilla.redhat.com/2318923",
            "https://bugzilla.redhat.com/2318925",
            "https://bugzilla.redhat.com/2318926",
            "https://bugzilla.redhat.com/2318927",
            "https://bugzilla.redhat.com/2331191",
            "https://bugzilla.redhat.com/2339218",
            "https://bugzilla.redhat.com/2339220",
            "https://bugzilla.redhat.com/2339221",
            "https://bugzilla.redhat.com/2339226",
            "https://bugzilla.redhat.com/2339231",
            "https://bugzilla.redhat.com/2339236",
            "https://bugzilla.redhat.com/2339238",
            "https://bugzilla.redhat.com/2339243",
            "https://bugzilla.redhat.com/2339247",
            "https://bugzilla.redhat.com/2339252",
            "https://bugzilla.redhat.com/2339259",
            "https://bugzilla.redhat.com/2339266",
            "https://bugzilla.redhat.com/2339270",
            "https://bugzilla.redhat.com/2339271",
            "https://bugzilla.redhat.com/2339275",
            "https://bugzilla.redhat.com/2339277",
            "https://bugzilla.redhat.com/2339281",
            "https://bugzilla.redhat.com/2339284",
            "https://bugzilla.redhat.com/2339291",
            "https://bugzilla.redhat.com/2339293",
            "https://bugzilla.redhat.com/2339295",
            "https://bugzilla.redhat.com/2339299",
            "https://bugzilla.redhat.com/2339300",
            "https://bugzilla.redhat.com/2339304",
            "https://bugzilla.redhat.com/2339305",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2294676",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2294677",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-37370",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-37371",
            "https://errata.almalinux.org/9/ALSA-2025-1671.html",
            "https://errata.rockylinux.org/RLSA-2024:5312",
            "https://github.com/krb5/krb5/commit/55fbf435edbe2e92dd8101669b1ce7144bc96fef",
            "https://linux.oracle.com/cve/CVE-2024-37371.html",
            "https://linux.oracle.com/errata/ELSA-2025-1673.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-37371",
            "https://security.netapp.com/advisory/ntap-20241108-0009/",
            "https://ubuntu.com/security/notices/USN-6947-1",
            "https://web.mit.edu/kerberos/www/advisories/",
            "https://web.mit.edu/kerberos/www/krb5-1.21/",
            "https://www.cve.org/CVERecord?id=CVE-2024-37371",
            "https://www.oracle.com/security-alerts/cpujan2025.html#AppendixMSQL"
          ],
          "PublishedDate": "2024-06-28T23:15:11.603Z",
          "LastModifiedDate": "2025-03-13T21:15:40.323Z"
        },
        {
          "VulnerabilityID": "CVE-2024-37370",
          "PkgID": "krb5-libs@1.20.1-r1",
          "PkgName": "krb5-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/krb5-libs@1.20.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "8d0773d3c16b905d"
          },
          "InstalledVersion": "1.20.1-r1",
          "FixedVersion": "1.20.2-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-37370",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "krb5: GSS message token handling",
          "Description": "In MIT Kerberos 5 (aka krb5) before 1.21.3, an attacker can modify the plaintext Extra Count field of a confidential GSS krb5 wrap token, causing the unwrapped token to appear truncated to the application.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-345"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6166",
            "https://access.redhat.com/security/cve/CVE-2024-37370",
            "https://bugzilla.redhat.com/2294676",
            "https://bugzilla.redhat.com/2294677",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2294676",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2294677",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-37370",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-37371",
            "https://errata.almalinux.org/9/ALSA-2024-6166.html",
            "https://errata.rockylinux.org/RLSA-2024:5312",
            "https://github.com/krb5/krb5/commit/55fbf435edbe2e92dd8101669b1ce7144bc96fef",
            "https://linux.oracle.com/cve/CVE-2024-37370.html",
            "https://linux.oracle.com/errata/ELSA-2024-6166.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-37370",
            "https://security.netapp.com/advisory/ntap-20241108-0007/",
            "https://ubuntu.com/security/notices/USN-6947-1",
            "https://web.mit.edu/kerberos/www/advisories/",
            "https://web.mit.edu/kerberos/www/krb5-1.21/",
            "https://www.cve.org/CVERecord?id=CVE-2024-37370"
          ],
          "PublishedDate": "2024-06-28T22:15:02.293Z",
          "LastModifiedDate": "2025-03-13T19:15:45.213Z"
        },
        {
          "VulnerabilityID": "CVE-2023-5363",
          "PkgID": "libcrypto3@3.1.1-r1",
          "PkgName": "libcrypto3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libcrypto3@3.1.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "15a350a62a57ea9c"
          },
          "InstalledVersion": "3.1.1-r1",
          "FixedVersion": "3.1.4-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-5363",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Incorrect cipher key and IV length processing",
          "Description": "Issue summary: A bug has been identified in the processing of key and\ninitialisation vector (IV) lengths.  This can lead to potential truncation\nor overruns during the initialisation of some symmetric ciphers.\n\nImpact summary: A truncation in the IV can result in non-uniqueness,\nwhich could result in loss of confidentiality for some cipher modes.\n\nWhen calling EVP_EncryptInit_ex2(), EVP_DecryptInit_ex2() or\nEVP_CipherInit_ex2() the provided OSSL_PARAM array is processed after\nthe key and IV have been established.  Any alterations to the key length,\nvia the \"keylen\" parameter or the IV length, via the \"ivlen\" parameter,\nwithin the OSSL_PARAM array will not take effect as intended, potentially\ncausing truncation or overreading of these values.  The following ciphers\nand cipher modes are impacted: RC2, RC4, RC5, CCM, GCM and OCB.\n\nFor the CCM, GCM and OCB cipher modes, truncation of the IV can result in\nloss of confidentiality.  For example, when following NIST's SP 800-38D\nsection 8.2.1 guidance for constructing a deterministic IV for AES in\nGCM mode, truncation of the counter portion could lead to IV reuse.\n\nBoth truncations and overruns of the key and overruns of the IV will\nproduce incorrect results and could, in some cases, trigger a memory\nexception.  However, these issues are not currently assessed as security\ncritical.\n\nChanging the key and/or IV lengths is not considered to be a common operation\nand the vulnerable API was recently introduced. Furthermore it is likely that\napplication developers will have spotted this problem during testing since\ndecryption would fail unless both peers in the communication were similarly\nvulnerable. For these reasons we expect the probability of an application being\nvulnerable to this to be quite low. However if an application is vulnerable then\nthis issue is considered very serious. For these reasons we have assessed this\nissue as Moderate severity overall.\n\nThe OpenSSL SSL/TLS implementation is not affected by this issue.\n\nThe OpenSSL 3.0 and 3.1 FIPS providers are not affected by this because\nthe issue lies outside of the FIPS provider boundary.\n\nOpenSSL 3.1 and 3.0 are vulnerable to this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-684"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2023/10/24/1",
            "https://access.redhat.com/errata/RHSA-2024:0310",
            "https://access.redhat.com/security/cve/CVE-2023-5363",
            "https://bugzilla.redhat.com/2243839",
            "https://errata.almalinux.org/9/ALSA-2024-0310.html",
            "https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=0df40630850fb2740e6be6890bb905d3fc623b2d",
            "https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=5f69f5c65e483928c4b28ed16af6e5742929f1ee",
            "https://linux.oracle.com/cve/CVE-2023-5363.html",
            "https://linux.oracle.com/errata/ELSA-2024-12093.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-5363",
            "https://security.netapp.com/advisory/ntap-20231027-0010/",
            "https://security.netapp.com/advisory/ntap-20240201-0003/",
            "https://security.netapp.com/advisory/ntap-20240201-0004/",
            "https://ubuntu.com/security/notices/USN-6450-1",
            "https://www.cve.org/CVERecord?id=CVE-2023-5363",
            "https://www.debian.org/security/2023/dsa-5532",
            "https://www.openssl.org/news/secadv/20231024.txt"
          ],
          "PublishedDate": "2023-10-25T18:17:43.613Z",
          "LastModifiedDate": "2024-11-21T08:41:36.81Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6119",
          "PkgID": "libcrypto3@3.1.1-r1",
          "PkgName": "libcrypto3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libcrypto3@3.1.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "15a350a62a57ea9c"
          },
          "InstalledVersion": "3.1.1-r1",
          "FixedVersion": "3.1.7-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6119",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Possible denial of service in X.509 name checks",
          "Description": "Issue summary: Applications performing certificate name checks (e.g., TLS\nclients checking server certificates) may attempt to read an invalid memory\naddress resulting in abnormal termination of the application process.\n\nImpact summary: Abnormal termination of an application can a cause a denial of\nservice.\n\nApplications performing certificate name checks (e.g., TLS clients checking\nserver certificates) may attempt to read an invalid memory address when\ncomparing the expected name with an `otherName` subject alternative name of an\nX.509 certificate. This may result in an exception that terminates the\napplication program.\n\nNote that basic certificate chain validation (signatures, dates, ...) is not\naffected, the denial of service can occur only when the application also\nspecifies an expected DNS name, Email address or IP address.\n\nTLS servers rarely solicit client certificates, and even when they do, they\ngenerally don't perform a name check against a reference identifier (expected\nidentity), but rather extract the presented identity after checking the\ncertificate chain.  So TLS servers are generally not affected and the severity\nof the issue is Moderate.\n\nThe FIPS modules in 3.3, 3.2, 3.1 and 3.0 are not affected by this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-843"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/09/03/4",
            "https://access.redhat.com/errata/RHSA-2024:8935",
            "https://access.redhat.com/security/cve/CVE-2024-6119",
            "https://bugzilla.redhat.com/2306158",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2306158",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6119",
            "https://errata.almalinux.org/9/ALSA-2024-8935.html",
            "https://errata.rockylinux.org/RLSA-2024:6783",
            "https://github.com/openssl/openssl/commit/05f360d9e849a1b277db628f1f13083a7f8dd04f",
            "https://github.com/openssl/openssl/commit/06d1dc3fa96a2ba5a3e22735a033012aadc9f0d6",
            "https://github.com/openssl/openssl/commit/621f3729831b05ee828a3203eddb621d014ff2b2",
            "https://github.com/openssl/openssl/commit/7dfcee2cd2a63b2c64b9b4b0850be64cb695b0a0",
            "https://github.com/openssl/openssl/security/advisories/GHSA-5qrj-vq78-58fj",
            "https://linux.oracle.com/cve/CVE-2024-6119.html",
            "https://linux.oracle.com/errata/ELSA-2024-8935.html",
            "https://lists.freebsd.org/archives/freebsd-security/2024-September/000303.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6119",
            "https://openssl-library.org/news/secadv/20240903.txt",
            "https://security.netapp.com/advisory/ntap-20240912-0001/",
            "https://ubuntu.com/security/notices/USN-6986-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6119"
          ],
          "PublishedDate": "2024-09-03T16:15:07.177Z",
          "LastModifiedDate": "2025-06-03T10:51:54.117Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45491",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45491",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: Integer Overflow or Wraparound",
          "Description": "An issue was discovered in libexpat before 2.6.3. dtdCopy in xmlparse.c can have an integer overflow for nDefaultAtts on 32-bit platforms (where UINT_MAX equals SIZE_MAX).",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 2,
            "photon": 4,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45491",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/888",
            "https://github.com/libexpat/libexpat/pull/891",
            "https://linux.oracle.com/cve/CVE-2024-45491.html",
            "https://linux.oracle.com/errata/ELSA-2024-8859.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45491",
            "https://security.netapp.com/advisory/ntap-20241018-0003/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://ubuntu.com/security/notices/USN-7001-1",
            "https://ubuntu.com/security/notices/USN-7001-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45491"
          ],
          "PublishedDate": "2024-08-30T03:15:03.85Z",
          "LastModifiedDate": "2024-11-21T09:37:51.08Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45492",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45492",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: integer overflow",
          "Description": "An issue was discovered in libexpat before 2.6.3. nextScaffoldPart in xmlparse.c can have an integer overflow for m_groupSize on 32-bit platforms (where UINT_MAX equals SIZE_MAX).",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 2,
            "photon": 4,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 6.2
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45492",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/889",
            "https://github.com/libexpat/libexpat/pull/892",
            "https://linux.oracle.com/cve/CVE-2024-45492.html",
            "https://linux.oracle.com/errata/ELSA-2024-6989.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45492",
            "https://security.netapp.com/advisory/ntap-20241018-0005/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45492"
          ],
          "PublishedDate": "2024-08-30T03:15:03.93Z",
          "LastModifiedDate": "2024-11-21T09:37:51.307Z"
        },
        {
          "VulnerabilityID": "CVE-2023-52425",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.6.0-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-52425",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "expat: parsing large tokens can trigger a denial of service",
          "Description": "libexpat through 2.5.0 allows a denial of service (resource consumption) because many full reparsings are required in the case of a large token for which multiple buffer fills are needed.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-400"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/03/20/5",
            "https://access.redhat.com/errata/RHSA-2024:1530",
            "https://access.redhat.com/security/cve/CVE-2023-52425",
            "https://bugzilla.redhat.com/2262877",
            "https://bugzilla.redhat.com/2268766",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2262877",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-52425",
            "https://errata.almalinux.org/9/ALSA-2024-1530.html",
            "https://errata.rockylinux.org/RLSA-2024:1615",
            "https://github.com/libexpat/libexpat/pull/789",
            "https://linux.oracle.com/cve/CVE-2023-52425.html",
            "https://linux.oracle.com/errata/ELSA-2024-4259.html",
            "https://lists.debian.org/debian-lts-announce/2024/04/msg00006.html",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/PNRIHC7DVVRAIWFRGV23Y6UZXFBXSQDB/",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/WNUBSGZFEZOBHJFTAD42SAN4ATW2VEMV/",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-52425",
            "https://security.netapp.com/advisory/ntap-20240614-0003/",
            "https://ubuntu.com/security/notices/USN-6694-1",
            "https://www.cve.org/CVERecord?id=CVE-2023-52425"
          ],
          "PublishedDate": "2024-02-04T20:15:46.063Z",
          "LastModifiedDate": "2024-11-21T08:39:43.393Z"
        },
        {
          "VulnerabilityID": "CVE-2024-28757",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.6.2-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-28757",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "expat: XML Entity Expansion",
          "Description": "libexpat through 2.6.1 allows an XML Entity Expansion attack when there is isolated use of external parsers (created via XML_ExternalEntityParserCreate).",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-776"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/03/15/1",
            "https://access.redhat.com/errata/RHSA-2024:1530",
            "https://access.redhat.com/security/cve/CVE-2024-28757",
            "https://bugzilla.redhat.com/2262877",
            "https://bugzilla.redhat.com/2268766",
            "https://errata.almalinux.org/9/ALSA-2024-1530.html",
            "https://github.com/libexpat/libexpat/issues/839",
            "https://github.com/libexpat/libexpat/pull/842",
            "https://linux.oracle.com/cve/CVE-2024-28757.html",
            "https://linux.oracle.com/errata/ELSA-2024-1530.html",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/FPLC6WDSRDUYS7F7JWAOVOHFNOUQ43DD/",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/LKJ7V5F6LJCEQJXDBWGT27J7NAP3E3N7/",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/VK2O34GH43NTHBZBN7G5Y6YKJKPUCTBE/",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-28757",
            "https://security.netapp.com/advisory/ntap-20240322-0001/",
            "https://ubuntu.com/security/notices/USN-6694-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-28757"
          ],
          "PublishedDate": "2024-03-10T05:15:06.57Z",
          "LastModifiedDate": "2025-03-28T19:15:21.23Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45490",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45490",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: Negative Length Parsing Vulnerability in libexpat",
          "Description": "An issue was discovered in libexpat before 2.6.3. xmlparse.c does not reject a negative length for XML_ParseBuffer.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-611",
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45490",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/887",
            "https://github.com/libexpat/libexpat/pull/890",
            "https://linux.oracle.com/cve/CVE-2024-45490.html",
            "https://linux.oracle.com/errata/ELSA-2024-6989.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45490",
            "https://security.netapp.com/advisory/ntap-20241018-0004/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://ubuntu.com/security/notices/USN-7001-1",
            "https://ubuntu.com/security/notices/USN-7001-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45490"
          ],
          "PublishedDate": "2024-08-30T03:15:03.757Z",
          "LastModifiedDate": "2025-03-14T19:15:47.253Z"
        },
        {
          "VulnerabilityID": "CVE-2024-8176",
          "PkgID": "libexpat@2.5.0-r1",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.5.0-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "52c504738aed8fed"
          },
          "InstalledVersion": "2.5.0-r1",
          "FixedVersion": "2.7.0-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-8176",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: expat: Improper Restriction of XML Entity Expansion Depth in libexpat",
          "Description": "A stack overflow vulnerability exists in the libexpat library due to the way it handles recursive entity expansion in XML documents. When parsing an XML document with deeply nested entity references, libexpat can be forced to recurse indefinitely, exhausting the stack space and causing a crash. This issue could lead to denial of service (DoS) or, in some cases, exploitable memory corruption, depending on the environment and library usage.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-674"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/03/15/1",
            "https://access.redhat.com/errata/RHSA-2025:13681",
            "https://access.redhat.com/errata/RHSA-2025:3531",
            "https://access.redhat.com/errata/RHSA-2025:3734",
            "https://access.redhat.com/errata/RHSA-2025:3913",
            "https://access.redhat.com/errata/RHSA-2025:4048",
            "https://access.redhat.com/errata/RHSA-2025:4446",
            "https://access.redhat.com/errata/RHSA-2025:4447",
            "https://access.redhat.com/errata/RHSA-2025:4448",
            "https://access.redhat.com/errata/RHSA-2025:4449",
            "https://access.redhat.com/errata/RHSA-2025:7444",
            "https://access.redhat.com/errata/RHSA-2025:7512",
            "https://access.redhat.com/errata/RHSA-2025:8385",
            "https://access.redhat.com/security/cve/CVE-2024-8176",
            "https://blog.hartwork.org/posts/expat-2-7-0-released/",
            "https://bugzilla.redhat.com/2310137",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2310137",
            "https://bugzilla.suse.com/show_bug.cgi?id=1239618",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-8176",
            "https://errata.almalinux.org/9/ALSA-2025-7444.html",
            "https://errata.rockylinux.org/RLSA-2025:4048",
            "https://github.com/libexpat/libexpat/blob/R_2_7_0/expat/Changes#L40-L52",
            "https://github.com/libexpat/libexpat/issues/893",
            "https://gitlab.alpinelinux.org/alpine/aports/-/commit/d068c3ff36fc6f4789988a09c69b434db757db53",
            "https://linux.oracle.com/cve/CVE-2024-8176.html",
            "https://linux.oracle.com/errata/ELSA-2025-7512.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-8176",
            "https://security-tracker.debian.org/tracker/CVE-2024-8176",
            "https://security.netapp.com/advisory/ntap-20250328-0009/",
            "https://ubuntu.com/security/CVE-2024-8176",
            "https://ubuntu.com/security/notices/USN-7424-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-8176",
            "https://www.kb.cert.org/vuls/id/760160"
          ],
          "PublishedDate": "2025-03-14T09:15:14.157Z",
          "LastModifiedDate": "2025-08-14T16:15:31.497Z"
        },
        {
          "VulnerabilityID": "CVE-2023-5363",
          "PkgID": "libssl3@3.1.1-r1",
          "PkgName": "libssl3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libssl3@3.1.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "1f546d0850661534"
          },
          "InstalledVersion": "3.1.1-r1",
          "FixedVersion": "3.1.4-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-5363",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Incorrect cipher key and IV length processing",
          "Description": "Issue summary: A bug has been identified in the processing of key and\ninitialisation vector (IV) lengths.  This can lead to potential truncation\nor overruns during the initialisation of some symmetric ciphers.\n\nImpact summary: A truncation in the IV can result in non-uniqueness,\nwhich could result in loss of confidentiality for some cipher modes.\n\nWhen calling EVP_EncryptInit_ex2(), EVP_DecryptInit_ex2() or\nEVP_CipherInit_ex2() the provided OSSL_PARAM array is processed after\nthe key and IV have been established.  Any alterations to the key length,\nvia the \"keylen\" parameter or the IV length, via the \"ivlen\" parameter,\nwithin the OSSL_PARAM array will not take effect as intended, potentially\ncausing truncation or overreading of these values.  The following ciphers\nand cipher modes are impacted: RC2, RC4, RC5, CCM, GCM and OCB.\n\nFor the CCM, GCM and OCB cipher modes, truncation of the IV can result in\nloss of confidentiality.  For example, when following NIST's SP 800-38D\nsection 8.2.1 guidance for constructing a deterministic IV for AES in\nGCM mode, truncation of the counter portion could lead to IV reuse.\n\nBoth truncations and overruns of the key and overruns of the IV will\nproduce incorrect results and could, in some cases, trigger a memory\nexception.  However, these issues are not currently assessed as security\ncritical.\n\nChanging the key and/or IV lengths is not considered to be a common operation\nand the vulnerable API was recently introduced. Furthermore it is likely that\napplication developers will have spotted this problem during testing since\ndecryption would fail unless both peers in the communication were similarly\nvulnerable. For these reasons we expect the probability of an application being\nvulnerable to this to be quite low. However if an application is vulnerable then\nthis issue is considered very serious. For these reasons we have assessed this\nissue as Moderate severity overall.\n\nThe OpenSSL SSL/TLS implementation is not affected by this issue.\n\nThe OpenSSL 3.0 and 3.1 FIPS providers are not affected by this because\nthe issue lies outside of the FIPS provider boundary.\n\nOpenSSL 3.1 and 3.0 are vulnerable to this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-684"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2023/10/24/1",
            "https://access.redhat.com/errata/RHSA-2024:0310",
            "https://access.redhat.com/security/cve/CVE-2023-5363",
            "https://bugzilla.redhat.com/2243839",
            "https://errata.almalinux.org/9/ALSA-2024-0310.html",
            "https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=0df40630850fb2740e6be6890bb905d3fc623b2d",
            "https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=5f69f5c65e483928c4b28ed16af6e5742929f1ee",
            "https://linux.oracle.com/cve/CVE-2023-5363.html",
            "https://linux.oracle.com/errata/ELSA-2024-12093.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-5363",
            "https://security.netapp.com/advisory/ntap-20231027-0010/",
            "https://security.netapp.com/advisory/ntap-20240201-0003/",
            "https://security.netapp.com/advisory/ntap-20240201-0004/",
            "https://ubuntu.com/security/notices/USN-6450-1",
            "https://www.cve.org/CVERecord?id=CVE-2023-5363",
            "https://www.debian.org/security/2023/dsa-5532",
            "https://www.openssl.org/news/secadv/20231024.txt"
          ],
          "PublishedDate": "2023-10-25T18:17:43.613Z",
          "LastModifiedDate": "2024-11-21T08:41:36.81Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6119",
          "PkgID": "libssl3@3.1.1-r1",
          "PkgName": "libssl3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libssl3@3.1.1-r1?arch=x86_64\u0026distro=3.18.2",
            "UID": "1f546d0850661534"
          },
          "InstalledVersion": "3.1.1-r1",
          "FixedVersion": "3.1.7-r0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6119",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Possible denial of service in X.509 name checks",
          "Description": "Issue summary: Applications performing certificate name checks (e.g., TLS\nclients checking server certificates) may attempt to read an invalid memory\naddress resulting in abnormal termination of the application process.\n\nImpact summary: Abnormal termination of an application can a cause a denial of\nservice.\n\nApplications performing certificate name checks (e.g., TLS clients checking\nserver certificates) may attempt to read an invalid memory address when\ncomparing the expected name with an `otherName` subject alternative name of an\nX.509 certificate. This may result in an exception that terminates the\napplication program.\n\nNote that basic certificate chain validation (signatures, dates, ...) is not\naffected, the denial of service can occur only when the application also\nspecifies an expected DNS name, Email address or IP address.\n\nTLS servers rarely solicit client certificates, and even when they do, they\ngenerally don't perform a name check against a reference identifier (expected\nidentity), but rather extract the presented identity after checking the\ncertificate chain.  So TLS servers are generally not affected and the severity\nof the issue is Moderate.\n\nThe FIPS modules in 3.3, 3.2, 3.1 and 3.0 are not affected by this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-843"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/09/03/4",
            "https://access.redhat.com/errata/RHSA-2024:8935",
            "https://access.redhat.com/security/cve/CVE-2024-6119",
            "https://bugzilla.redhat.com/2306158",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2306158",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6119",
            "https://errata.almalinux.org/9/ALSA-2024-8935.html",
            "https://errata.rockylinux.org/RLSA-2024:6783",
            "https://github.com/openssl/openssl/commit/05f360d9e849a1b277db628f1f13083a7f8dd04f",
            "https://github.com/openssl/openssl/commit/06d1dc3fa96a2ba5a3e22735a033012aadc9f0d6",
            "https://github.com/openssl/openssl/commit/621f3729831b05ee828a3203eddb621d014ff2b2",
            "https://github.com/openssl/openssl/commit/7dfcee2cd2a63b2c64b9b4b0850be64cb695b0a0",
            "https://github.com/openssl/openssl/security/advisories/GHSA-5qrj-vq78-58fj",
            "https://linux.oracle.com/cve/CVE-2024-6119.html",
            "https://linux.oracle.com/errata/ELSA-2024-8935.html",
            "https://lists.freebsd.org/archives/freebsd-security/2024-September/000303.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6119",
            "https://openssl-library.org/news/secadv/20240903.txt",
            "https://security.netapp.com/advisory/ntap-20240912-0001/",
            "https://ubuntu.com/security/notices/USN-6986-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6119"
          ],
          "PublishedDate": "2024-09-03T16:15:07.177Z",
          "LastModifiedDate": "2025-06-03T10:51:54.117Z"
        },
        {
          "VulnerabilityID": "CVE-2023-7104",
          "PkgID": "sqlite-libs@3.41.2-r2",
          "PkgName": "sqlite-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/sqlite-libs@3.41.2-r2?arch=x86_64\u0026distro=3.18.2",
            "UID": "96051f548b6df5e4"
          },
          "InstalledVersion": "3.41.2-r2",
          "FixedVersion": "3.41.2-r3",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-7104",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "sqlite: heap-buffer-overflow at sessionfuzz",
          "Description": "A vulnerability was found in SQLite SQLite3 up to 3.43.0 and classified as critical. This issue affects the function sessionReadRecord of the file ext/session/sqlite3session.c of the component make alltest Handler. The manipulation leads to heap-based buffer overflow. It is recommended to apply a patch to fix this issue. The associated identifier of this vulnerability is VDB-248999.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-122",
            "CWE-119"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L",
              "V3Score": 7.3
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L",
              "V3Score": 7.3
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L",
              "V3Score": 7.3
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:0465",
            "https://access.redhat.com/security/cve/CVE-2023-7104",
            "https://bugzilla.redhat.com/2256194",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2256194",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-7104",
            "https://errata.almalinux.org/9/ALSA-2024-0465.html",
            "https://errata.rockylinux.org/RLSA-2024:0253",
            "https://linux.oracle.com/cve/CVE-2023-7104.html",
            "https://linux.oracle.com/errata/ELSA-2024-0465.html",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/AYONA2XSNFMXLAW4IHLFI5UVV3QRNG5K/",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/D6C2HN4T2S6GYNTAUXLH45LQZHK7QPHP/",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-7104",
            "https://security.netapp.com/advisory/ntap-20240112-0008/",
            "https://sqlite.org/forum/forumpost/5bcbf4571c",
            "https://sqlite.org/src/info/0e4e7a05c4204b47",
            "https://ubuntu.com/security/notices/USN-6566-1",
            "https://ubuntu.com/security/notices/USN-6566-2",
            "https://vuldb.com/?ctiid.248999",
            "https://vuldb.com/?id.248999",
            "https://www.cve.org/CVERecord?id=CVE-2023-7104"
          ],
          "PublishedDate": "2023-12-29T10:15:13.89Z",
          "LastModifiedDate": "2024-11-21T08:45:17.067Z"
        },
        {
          "VulnerabilityID": "CVE-2022-48174",
          "PkgID": "ssl_client@1.36.1-r0",
          "PkgName": "ssl_client",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/ssl_client@1.36.1-r0?arch=x86_64\u0026distro=3.18.2",
            "UID": "582fb58b08117b88"
          },
          "InstalledVersion": "1.36.1-r0",
          "FixedVersion": "1.36.1-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:31e352740f534f9ad170f75378a84fe453d6156e40700b882d737a8f4a6988a3",
            "DiffID": "sha256:78a822fe2a2d2c84f3de4a403188c45f623017d6a4521d23047c9fbb0801794c"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2022-48174",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "busybox: stack overflow vulnerability in ash.c leads to arbitrary code execution",
          "Description": "There is a stack overflow vulnerability in ash.c:6030 in busybox before 1.35. In the environment of Internet of Vehicles, this vulnerability can be executed from command to arbitrary code execution.",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-787"
          ],
          "VendorSeverity": {
            "amazon": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 3,
            "redhat": 3,
            "ubuntu": 1
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2022-48174",
            "https://bugs.busybox.net/show_bug.cgi?id=15216",
            "https://linux.oracle.com/cve/CVE-2022-48174.html",
            "https://linux.oracle.com/errata/ELSA-2023-5178.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2022-48174",
            "https://security.netapp.com/advisory/ntap-20241129-0001/",
            "https://ubuntu.com/security/notices/USN-6335-1",
            "https://ubuntu.com/security/notices/USN-6961-1",
            "https://www.cve.org/CVERecord?id=CVE-2022-48174"
          ],
          "PublishedDate": "2023-08-22T19:16:31.08Z",
          "LastModifiedDate": "2025-02-05T18:02:49.267Z"
        },
        {
          "VulnerabilityID": "CVE-2025-31115",
          "PkgID": "xz-libs@5.4.3-r0",
          "PkgName": "xz-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/xz-libs@5.4.3-r0?arch=x86_64\u0026distro=3.18.2",
            "UID": "bfe5d1a66098ff1a"
          },
          "InstalledVersion": "5.4.3-r0",
          "FixedVersion": "5.4.3-r1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:8455851b3630dbcb4a652ff07c9acec95d938e3e0d4ed5f33ccdf8b0d19ac5eb",
            "DiffID": "sha256:9df700eb1eef3549a281b500c89f03ca9455f9b4661755a001b1c15b4a90b588"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-31115",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "xz: XZ has a heap-use-after-free bug in threaded .xz decoder",
          "Description": "XZ Utils provide a general-purpose data-compression library plus command-line tools. In XZ Utils 5.3.3alpha to 5.8.0, the multithreaded .xz decoder in liblzma has a bug where invalid input can at least result in a crash. The effects include heap use after free and writing to an address based on the null pointer plus an offset. Applications and libraries that use the lzma_stream_decoder_mt function are affected. The bug has been fixed in XZ Utils 5.8.1, and the fix has been committed to the v5.4, v5.6, v5.8, and master branches in the xz Git repository. No new release packages will be made from the old stable branches, but a standalone patch is available that applies to all affected releases.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-366",
            "CWE-416",
            "CWE-476",
            "CWE-826"
          ],
          "VendorSeverity": {
            "alma": 3,
            "azure": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/04/03/1",
            "http://www.openwall.com/lists/oss-security/2025/04/03/2",
            "http://www.openwall.com/lists/oss-security/2025/04/03/3",
            "https://access.redhat.com/errata/RHSA-2025:7524",
            "https://access.redhat.com/security/cve/CVE-2025-31115",
            "https://bugzilla.redhat.com/2357249",
            "https://errata.almalinux.org/10/ALSA-2025-7524.html",
            "https://github.com/tukaani-project/xz/commit/d5a2ffe41bb77b918a8c96084885d4dbe4bf6480",
            "https://github.com/tukaani-project/xz/security/advisories/GHSA-6cc8-p5mm-29w2",
            "https://linux.oracle.com/cve/CVE-2025-31115.html",
            "https://linux.oracle.com/errata/ELSA-2025-7524.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-31115",
            "https://tukaani.org/xz/xz-cve-2025-31115.patch",
            "https://ubuntu.com/security/notices/USN-7414-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-31115"
          ],
          "PublishedDate": "2025-04-03T17:15:30.54Z",
          "LastModifiedDate": "2025-04-07T14:18:34.453Z"
        }
      ]
    },
    {
      "Target": "Python",
      "Class": "lang-pkgs",
      "Type": "python-pkg",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2022-40897",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.7/site-packages/setuptools-57.5.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@57.5.0",
            "UID": "45a32bfca3f91ccf"
          },
          "InstalledVersion": "57.5.0",
          "FixedVersion": "65.5.1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:f053299a814f964803234a48003d7c807f15a04ab5d15c3dac974b2b9ac19f88",
            "DiffID": "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2022-40897",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa-setuptools: Regular Expression Denial of Service (ReDoS) in package_index.py",
          "Description": "Python Packaging Authority (PyPA) setuptools before 65.5.1 allows remote attackers to cause a denial of service via HTML in a crafted package or custom PackageIndex page. There is a Regular Expression Denial of Service (ReDoS) in package_index.py.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-1333"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "bitnami": 2,
            "cbl-mariner": 2,
            "ghsa": 3,
            "nvd": 2,
            "oracle-oval": 2,
            "photon": 2,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2023:0952",
            "https://access.redhat.com/security/cve/CVE-2022-40897",
            "https://bugzilla.redhat.com/2158559",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2158559",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40897",
            "https://errata.almalinux.org/9/ALSA-2023-0952.html",
            "https://errata.rockylinux.org/RLSA-2023:0952",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2022-43012.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/fe8a98e696241487ba6ac9f91faa38ade939ec5d/setuptools/package_index.py#L200",
            "https://github.com/pypa/setuptools/commit/43a9c9bfa6aa626ec2a22540bea28d2ca77964be",
            "https://github.com/pypa/setuptools/compare/v65.5.0...v65.5.1",
            "https://github.com/pypa/setuptools/issues/3659",
            "https://linux.oracle.com/cve/CVE-2022-40897.html",
            "https://linux.oracle.com/errata/ELSA-2024-2987.html",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ADES3NLOE5QJKBLGNZNI2RGVOSQXA37R",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ADES3NLOE5QJKBLGNZNI2RGVOSQXA37R/",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/YNA2BAH2ACBZ4TVJZKFLCR7L23BG5C3H",
            "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/YNA2BAH2ACBZ4TVJZKFLCR7L23BG5C3H/",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ADES3NLOE5QJKBLGNZNI2RGVOSQXA37R",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YNA2BAH2ACBZ4TVJZKFLCR7L23BG5C3H",
            "https://nvd.nist.gov/vuln/detail/CVE-2022-40897",
            "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages",
            "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
            "https://pyup.io/vulnerabilities/CVE-2022-40897/52495",
            "https://pyup.io/vulnerabilities/CVE-2022-40897/52495/",
            "https://security.netapp.com/advisory/ntap-20230214-0001",
            "https://security.netapp.com/advisory/ntap-20230214-0001/",
            "https://security.netapp.com/advisory/ntap-20240621-0006",
            "https://security.netapp.com/advisory/ntap-20240621-0006/",
            "https://setuptools.pypa.io/en/latest",
            "https://ubuntu.com/security/notices/USN-5817-1",
            "https://www.cve.org/CVERecord?id=CVE-2022-40897"
          ],
          "PublishedDate": "2022-12-23T00:15:13.987Z",
          "LastModifiedDate": "2024-11-21T07:22:13.787Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.7/site-packages/setuptools-57.5.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@57.5.0",
            "UID": "45a32bfca3f91ccf"
          },
          "InstalledVersion": "57.5.0",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:f053299a814f964803234a48003d7c807f15a04ab5d15c3dac974b2b9ac19f88",
            "DiffID": "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:6726",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.7/site-packages/setuptools-57.5.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@57.5.0",
            "UID": "45a32bfca3f91ccf"
          },
          "InstalledVersion": "57.5.0",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:f053299a814f964803234a48003d7c807f15a04ab5d15c3dac974b2b9ac19f88",
            "DiffID": "sha256:709bcbb5a424f97a0b7bde73b9bf66e33cbbb8d375940c553b2f7e8af5f4d13a"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "root/.local/share/virtualenv/wheel/3.7/image/1/CopyPipInstall/setuptools-68.0.0-py3-none-any/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "5e6ed1cfc1fd6551"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
            "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:6726",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.7/site-packages/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "53f816cb2e747bba"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
            "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:6726",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.7/site-packages/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "e93a2f114480bceb"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:2a88c5c4b5ec4f7721b26e9fdbf73049eb2ec929611bb72deb12b830f452a738",
            "DiffID": "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:6726",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "root/.local/share/virtualenv/wheel/3.7/image/1/CopyPipInstall/setuptools-68.0.0-py3-none-any/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "5e6ed1cfc1fd6551"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
            "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.7/site-packages/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "53f816cb2e747bba"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
            "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.7/site-packages/setuptools-68.0.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@68.0.0",
            "UID": "e93a2f114480bceb"
          },
          "InstalledVersion": "68.0.0",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:2a88c5c4b5ec4f7721b26e9fdbf73049eb2ec929611bb72deb12b830f452a738",
            "DiffID": "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        },
        {
          "VulnerabilityID": "CVE-2023-43804",
          "PkgName": "urllib3",
          "PkgPath": "root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.7/site-packages/urllib3-2.0.4.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/urllib3@2.0.4",
            "UID": "732d4d02084bf943"
          },
          "InstalledVersion": "2.0.4",
          "FixedVersion": "2.0.6, 1.26.17",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:6e6c998f67475b04d0d3fa2cb594d1961dea1fc9bbf46be814417be6e2d1a2b3",
            "DiffID": "sha256:8fe138ff26c8cf6f16cde828de7604b06b9175b5323f7b949884fad811a16207"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2023-43804",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "python-urllib3: Cookie request header isn't stripped during cross-origin redirects",
          "Description": "urllib3 is a user-friendly HTTP client library for Python. urllib3 doesn't treat the `Cookie` HTTP header special or provide any helpers for managing cookies over HTTP, that is the responsibility of the user. However, it is possible for a user to specify a `Cookie` header and unknowingly leak information via HTTP redirects to a different origin if that user doesn't disable redirects explicitly. This issue has been patched in urllib3 version 1.26.17 or 2.0.5.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-200"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 2,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:N",
              "V3Score": 5.9
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:N",
              "V3Score": 8.1
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:N",
              "V3Score": 5.9
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:2159",
            "https://access.redhat.com/security/cve/CVE-2023-43804",
            "https://bugzilla.redhat.com/2242493",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2242493",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-43804",
            "https://errata.almalinux.org/9/ALSA-2024-2159.html",
            "https://errata.rockylinux.org/RLSA-2024:2986",
            "https://github.com/pypa/advisory-database/tree/main/vulns/urllib3/PYSEC-2023-192.yaml",
            "https://github.com/urllib3/urllib3",
            "https://github.com/urllib3/urllib3/commit/01220354d389cd05474713f8c982d05c9b17aafb",
            "https://github.com/urllib3/urllib3/commit/644124ecd0b6e417c527191f866daa05a5a2056d",
            "https://github.com/urllib3/urllib3/security/advisories/GHSA-v845-jxx5-vc9f",
            "https://linux.oracle.com/cve/CVE-2023-43804.html",
            "https://linux.oracle.com/errata/ELSA-2024-2987.html",
            "https://lists.debian.org/debian-lts-announce/2023/10/msg00012.html",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/5F5CUBAN5XMEBVBZPHFITBLMJV5FIJJ5",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/5F5CUBAN5XMEBVBZPHFITBLMJV5FIJJ5/",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/I3PR7C6RJ6JUBQKIJ644DMIJSUP36VDY",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/I3PR7C6RJ6JUBQKIJ644DMIJSUP36VDY/",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NDAGZXYJ7H2G3SB47M453VQVNAWKAEJJ",
            "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NDAGZXYJ7H2G3SB47M453VQVNAWKAEJJ/",
            "https://nvd.nist.gov/vuln/detail/CVE-2023-43804",
            "https://security.netapp.com/advisory/ntap-20241213-0007",
            "https://security.netapp.com/advisory/ntap-20241213-0007/",
            "https://ubuntu.com/security/notices/USN-6473-1",
            "https://ubuntu.com/security/notices/USN-6473-2",
            "https://www.cve.org/CVERecord?id=CVE-2023-43804",
            "https://www.vicarius.io/vsociety/posts/cve-2023-43804-urllib3-vulnerability-3"
          ],
          "PublishedDate": "2023-10-04T17:15:10.163Z",
          "LastModifiedDate": "2024-12-13T14:15:20.57Z"
        },
        {
          "VulnerabilityID": "CVE-2024-53899",
          "PkgName": "virtualenv",
          "PkgPath": "usr/local/lib/python3.7/site-packages/virtualenv-20.24.2.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/virtualenv@20.24.2",
            "UID": "bc5b01e428c1f76b"
          },
          "InstalledVersion": "20.24.2",
          "FixedVersion": "20.26.6",
          "Status": "fixed",
          "Layer": {
            "Digest": "sha256:2a88c5c4b5ec4f7721b26e9fdbf73049eb2ec929611bb72deb12b830f452a738",
            "DiffID": "sha256:2b60d6851c91d8483023b98a1a4d0051108a0d94d014e776f1295d5f408738de"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-53899",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "virtualenv: potential command injection via virtual environment activation scripts",
          "Description": "virtualenv before 20.26.6 allows command injection through the activation scripts for a virtual environment. Magic template strings are not quoted correctly when replacing. NOTE: this is not the same as CVE-2024-9287.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-77",
            "CWE-78"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.4
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 7.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:10953",
            "https://access.redhat.com/security/cve/CVE-2024-53899",
            "https://bugzilla.redhat.com/2328554",
            "https://errata.almalinux.org/8/ALSA-2024-10953.html",
            "https://github.com/pypa/advisory-database/tree/main/vulns/virtualenv/PYSEC-2024-187.yaml",
            "https://github.com/pypa/virtualenv",
            "https://github.com/pypa/virtualenv/issues/2768",
            "https://github.com/pypa/virtualenv/pull/2771",
            "https://github.com/pypa/virtualenv/releases/tag/20.26.6",
            "https://linux.oracle.com/cve/CVE-2024-53899.html",
            "https://linux.oracle.com/errata/ELSA-2024-11048.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-53899",
            "https://ubuntu.com/security/notices/USN-7271-1",
            "https://ubuntu.com/security/notices/USN-7271-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-53899"
          ],
          "PublishedDate": "2024-11-24T16:15:06.647Z",
          "LastModifiedDate": "2025-02-10T18:12:06.107Z"
        }
      ]
    }
  ]
}
